mpa/det/trainer.py

```diff
@@ -138,7 +138,7 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         best_ckpt_path = glob.glob(osp.join(cfg.work_dir, 'best_segm_mAP*.pth'))
         if len(best_ckpt_path) > 0:
             output_ckpt_path = best_ckpt_path[0]
-        best_ckpt_path = glob.glob(osp.join(cfg.work_dir, 'best_bbox_mAP*.pth'))
+        best_ckpt_path = glob.glob(osp.join(cfg.work_dir, 'best_mAP*.pth'))
```

Comment:
What do you think of using best_*.pth? Then we don't need to care about specific metrics.
@harimkang Would this idea work?

Reply:
As it is, detection and instance-segmentation use 'mAP' in common, so it is right to bring the weights, 
but I think it is necessary to modify it considering the change of the metric that goes into the save_best of evaluation.

Reply:
Would it be too complicated to designate the part where the model is stored as 'best.pth' in the evaluation?

Reply:
So, @harimkang do you mean that best*.pth is proper for general situations?

Reply:
I meant that it should be 'best.pth' or 'f'best_{metric}*.pth', not 'best*.pth'.

Reply:
The actual file name for the best model is decided by mmdet not mpa; * is not only for metric but also epoch number. ex) It is being saved like best_mAP_epoch12.pth.

Reply:
Than the latter option(f'best_{metric}*.pth') is only available for better generalization. @goodsong81, What do you think?
I think both best_* and f'best_{metric}' could work